### PR TITLE
sql: does not support using ALTER TABLE to add columns with AUTO_INCREMENT

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

Does not support using ALTER TABLE to add columns with AUTO_INCREMENT

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v7.4 (TiDB 7.4 versions)
- [x] v7.3 (TiDB 7.3 versions)
- [x] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: auto-increment.md [hunk 1]
```diff
@@ -357,7 +357,7 @@ CREATE TABLE t(a int AUTO_INCREMENT key) AUTO_ID_CACHE 1;
 - 对于 v6.6.0 及更早的 TiDB 版本，定义的列必须为主键或者索引前缀。
 - 只能定义在类型为整数、`FLOAT` 或 `DOUBLE` 的列上。
 - 不支持与列的默认值 `DEFAULT` 同时指定在同一列上。
-- 不支持使用 `ALTER TABLE` 来添加 `AUTO_INCREMENT` 属性。
+- 不支持使用 `ALTER TABLE` 来添加带 `AUTO_INCREMENT` 属性的列。
```
Comment (hfxsd) [inline auto-increment.md:360]:
```suggestion
- 不支持使用 `ALTER TABLE` 来添加或修改带 `AUTO_INCREMENT` 属性的列。
```

Comment (bb7133) [reply auto-increment.md]: 不支持使用 `ALTER TABLE` 来添加 `AUTO_INCREMENT` 属性：包括使用 `ALTER TABLE ... MODIFY/CHANGE COLUMN` 语法为已存在的列增加 `AUTO_INCREMENT` 属性，以及使用 `ALTER TABLE ... ADD COLUMN` 增加带有 `AUTO_INCREMENT` 属性的列。

Comment (hfxsd) [inline auto-increment.md]:
```suggestion
- 不支持使用 `ALTER TABLE` 来添加 `AUTO_INCREMENT` 属性，包括使用 `ALTER TABLE ... MODIFY/CHANGE COLUMN` 语法为已存在的列添加 `AUTO_INCREMENT` 属性，以及使用 `ALTER TABLE ... ADD COLUMN` 添加带有 `AUTO_INCREMENT` 属性的列。
```

**Review by Defined2014 — APPROVED** [review]

**Review by bb7133 — APPROVED** [review]
LGTM

**Review by qiancai — APPROVED** [review]

**Pull request merged**